src/Raven.Studio/typescript/commands/database/tasks/testPeriodicBackupCredentialsCommand.ts

```diff
@@ -1,21 +1,19 @@
 import commandBase = require("commands/commandBase");
-import database = require("models/resources/database");
 import endpoints = require("endpoints");
 
 class testPeriodicBackupCredentialsCommand extends commandBase {
```

Comment:
should we move this command outside commands/database/tasks? it is no longer db related task.